index.html

```diff
@@ -14,11 +14,11 @@
         <div id="header" name="header">
 
             <div id="upperHeader" name="upperHeader">
-                <img src="../Gatham/assets/Logo2.png" id="mainLogo" name="mainLogo">
+                <img src="../GothamAutoRenov/assets/Logo2.png" id="mainLogo" name="mainLogo">
                 <input type="text" id="searchBox" name="searchBox" placeholder="   جستجو" size="20">
                 <div id="leftUpperHeader" name="leftUpperHeader">
                     <a href="login.html" id="loginLink">
-                        <img src="../Gatham/assets/signin.png" id="signImg">
+                        <img src="../GothamAutoRenov/assets/signin.png" id="signImg">
```

Comment:
the address you provided can be smaller without the name of container root folder